### PR TITLE
adding new fix 0127 for sort the list of device firmware updates

### PR DESCRIPTION
**Pull request by Maik2208**
The list of device firmware updates is now sorted by device-type and device-address

### Description
This fix will sort the list of available firmwareupdates for devices by the device-type and device-address.

### Related Issue
#1366 

### Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Possible Drawbacks
nothing known

### Verification Process
show the list of available firmwareupdates to check.

### Contributing checklist

- [X] My code follows the code style of this project.
- [X] I have read the **CONTRIBUTING** and **LICENSE** document.
- [X] I fully agree to distribute my changes under Apache 2.0 license.

### CODE REVIEW

File: buildroot-external/patches/occu/0127-WebUI-Sort-FirmwareOverview.patch [hunk 1]
```diff
@@ -0,0 +1,18 @@
+--- occu/WebUI/www/webui/webui.js.orig
++++ occu/WebUI/www/webui/webui.js
+@@ -20568,6 +20568,15 @@
+               }
+               conInfo("Check firmware version of all devices - number of devices: " + self.devList.length);
+               if (self.devList.length > 0) {
++								function sortorder_devList(a , b){
```
Comment (jens-maus) [inline buildroot-external/patches/occu/0127-WebUI-Sort-FirmwareOverview.patch:7]: Bitte hier keine Tabs, sondern Spaces verwenden. Auch bitte immer sich am jeweiligen Space/Tab Style der Datei orientieren und auch die Anzahl an Einrückungen exakt gleich machen wie das im rest der Datei so ist.

Comment (Maik2208) [reply buildroot-external/patches/occu/0127-WebUI-Sort-FirmwareOverview.patch:7]: Sorry, ist geändert

Comment (jens-maus) [reply buildroot-external/patches/occu/0127-WebUI-Sort-FirmwareOverview.patch:7]: Nein, ist nicht geändert in diesem PR.

Comment (Maik2208) [reply buildroot-external/patches/occu/0127-WebUI-Sort-FirmwareOverview.patch]: Okay, also man muss nach dem Commit auch noch Push drücken 🙈

**Pull request merged**